draft-ietf-openpgp-pqc.md

```diff
@@ -303,15 +303,17 @@ based on these mathematical problems is endangered by the advent of quantum
 computers, there is a need to extend OpenPGP by algorithms that remain secure
 in the presence of quantum computers.
 
-Such cryptographic algorithms are referred to as post-quantum cryptography.
-The algorithms defined in this extension were chosen for standardization by the
+Such cryptographic algorithms are referred to as post-quantum cryptography.  The
+algorithms defined in this extension were chosen for standardization by the
```

Comment:
this seems to just be a textual reflow.  is there a reason to include it in this change?

Reply:
It seems to have happened due to an inaccuracy of my editor (Vim) which made a reflow to limit the line to one character less than I specified. I reverted it.